tests/bluetooth/bt_crypto/bt-crypto-mbedtls.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_PSA_CRYPTO_C=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_MBEDTLS_ZEPHYR_ENTROPY=y
+
+CONFIG_MBEDTLS_TLS_VERSION_1_2=n
+CONFIG_MBEDTLS_KEY_EXCHANGE_RSA_ENABLED=n
+CONFIG_MBEDTLS_CIPHER_DES_ENABLED=n
+CONFIG_MBEDTLS_CIPHER_MODE_CBC_ENABLED=n
+CONFIG_MBEDTLS_PK_WRITE_C=n
```

Comment:
Make sure to remove those once https://github.com/zephyrproject-rtos/zephyr/pull/72475 is merged.

Reply:
Since #72475 was merged, I just rebased the PR removing these lines